daru-view.gemspec

```diff
@@ -36,7 +36,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rubocop'
 
   # fetching latest gem from the Gemfile
-  spec.add_runtime_dependency 'google_visualr'
+  spec.add_runtime_dependency 'google_visualr_rails5'
```

Comment:
Please let us know the what change make it work. So that we get to know if we really need to switch.